docs/content/preview/yugabyte-voyager/known-issues/postgresql.md

```diff
@@ -1461,4 +1465,128 @@ CREATE TABLE test_jsonb_chk (
     data1 jsonb,
     CHECK (data1->'key'<>'{}')
 );
-```
\ No newline at end of file
+```
+
+### Events Listen / Notify
+
+**GitHub**: Issue [#1872](https://github.com/yugabyte/yugabyte-db/issues/1872)
+
+**Description**: If your application queries or PL/pgSQL objects rely on **LISTEN/NOTIFY events** in the source PostgreSQL database, these functionalities will not work after migrating to YugabyteDB. Currently, LISTEN/NOTIFY events are a no-op in YugabyteDB, and any attempt to use them will trigger a warning instead of performing the expected event-driven operations:
+
+```sql
+WARNING:  LISTEN not supported yet and will be ignored
+```
+
+**Workaround**: Currently, there is no workaround.
+
+**Example:**
+
+```sql
+LISTEN my_table_changes;
+INSERT INTO my_table (name) VALUES ('Charlie');
+NOTIFY my_table_changes, 'New row added with name: Charlie';
+```
+
+### Two-Phase Commit
+
+**GitHub**: Issue [#11084](https://github.com/yugabyte/yugabyte-db/issues/11084)
+
+**Description**: If your application queries or PL/pgSQL objects rely on **Two-Phase Commit** protocol that allows multiple distributed systems to work together in a transactional manner in the [source](https://www.postgresql.org/docs/current/two-phase.html) PostgreSQL database, these functionalities will not work after migrating to YugabyteDB. Currently, Two-Phase Commit is not implemented in the YugabyteDB and will throw the following error when attempt to execute the commands:
+
+```sql
+ERROR:  PREPARE TRANSACTION not supported yet
+```
+
+**Workaround**: Currently, there is no workaround.
+
+## Limitations
+
+There are certain limitations when reporting issues in [assess-migration](../../reference/assess-migration/) and [analyze-schema](../../reference/schema-migration/analyze-schema/) commands:
+
+1. Normalized queries in `pg_stat_statements`
+
+    The `pg_stat_statements` extension in PostgreSQL tracks execution statistics of SQL queries by normalizing them. Since normalization removes the constants, the issues related to the constants for the following scenarios cannot be detected during assessment, and analyze:
+
+    - `JSON_TABLE` usage in DML statements.
+    - Non-decimal integer literals in DML statements.
+    - Two-Phase Commit (XA syntax) (Issue [#11084](https://github.com/yugabyte/yugabyte-db/issues/11084)).
+
+    **Example:**
+
+    ```sql
+    --query stored in pg_stat_statements:
+    SELECT $1 as binary; 
+
+    --actual query:
+    SELECT 0b101010 as binary;
+    ```
+
+1. Transactional Queries in `pg_stat_statements`
+
+    In `pg_stat_statements`, a single transaction is recorded as multiple separate query entries. This fragmentation makes it challenging to detect issues that occur within transaction boundaries, such as:
+
+    - DDL operations within Transaction (Issue [#1404](https://github.com/yugabyte/yugabyte-db/issues/1404)).
```

Comment:
change this point to `DDL operations within Transaction` and  link to the issue you will be adding as I shared in the doc